### ISSUE TITLE
pastehunter-cli execution

### ISSUE THREAD

**Issue by H4ckW0lf** [kevthehermit/PasteHunter#113]
Hello!

I hope they are very good, I am new using the tool. I did the installation with docker-compuse up and it correctly opens ELK, however when I want to run the tool as such from its / opt / PasteHunter directory with the following command:

root @ kali: / opt / PasteHunter # python3 pastehunter-cli
INFO: pastehunter-cli: Starting PasteHunter Version: 1.0
INFO: pastehunter-cli: Reading Configs
ERROR: common.py: Unable to parse config file: [Errno 21] Is a directory: '/root/.config/pastehunter.json'

I get the errors shown in the lines above.

If I do it only with python, I get this:

root @ kali: / opt / PasteHunter # python pastehunter-cli
Traceback (most recent call last):
   File "pastehunter-cli", line 15, in <module>
     from urllib.parse import unquote_plus
ImportError: No module named parse

Am I missing another step?

**Comment by H4ckW0lf** [comment]
I resolved my problem, I had not successfully installed pasthunter.json

mkdir -p ~ / .config
wget https://raw.githubusercontent.com/kevthehermit/PasteHunter/master/settings.json.sample -O ~ / .config / pastehunter.json

when I did it correctly it automatically runs when I do the docker-compose up

**Comment by youda313** [comment]
I have the issue while running the Docker-compose

pastehunter      | wait-for-it.sh: 172.16.10.10:9200 is available after 16 seconds           pastehunter      | INFO:pastehunter-cli:Starting PasteHunter Version: 1.4.0
pastehunter      | INFO:pastehunter-cli:Reading Configs
pastehunter      | ERROR:common.py:Unable to parse config file: [Errno 21] Is a directory: '/root/.config/pastehunter.json'
pastehunter exited with code 0

**Comment by youda313** [comment]
Following what was mentioned above I have added the pastehunter.json into the .config folder but I got now a new error:

Starting elasticsearch ...
Starting elasticsearch ... done
Starting kibana ...
Starting kibana ... done
Starting pastehunter ...
Starting pastehunter ... error

ERROR: for pastehunter  Cannot start service pastehunter: OCI runtime create failed: container_linux.go:380: starting container process caused: process_linux.go:545: container init caused: rootfs_linux.go:75: mounting "/home/user/.config/pastehunter.json" to rootfs at "/root/.config/pastehunter.json" caused: mount through procfd: not a directory: unknown: Are you trying to mount a directory onto a file (or vice-versa)? Check if the specified host path exists and is the expected type

ERROR: for pastehunter  Cannot start service pastehunter: OCI runtime create failed: container_linux.go:380: starting container process caused: process_linux.go:545: container init caused: rootfs_linux.go:75: mounting "/home/user/.config/pastehunter.json" to rootfs at "/root/.config/pastehunter.json" caused: mount through procfd: not a directory: unknown: Are you trying to mount a directory onto a file (or vice-versa)? Check if the specified host path exists and is the expected type
ERROR: Encountered errors while bringing up the project.

The error does not make any sense as I pastehunter.json is a file and not a folder
